src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -545,43 +545,39 @@ private boolean offerNotEnoughResources(Offer offer, double cpus, int mem) {
   }
 
   private void reconcileTasks(SchedulerDriver driver) {
-    updateReconciliationDeadline();
```

Comment:
Sure, will do. Each time the scheduler is instantiated, it will be set to false as that is default for booleans, but we should be more explicit about it.